apps/marketplace/components/SellerAssessment/SellerAssessmentCriteriaStage.js

```diff
@@ -20,6 +20,35 @@ const getCriteriaNeeded = (criteriaNeeded, priceMaximum, maxDailyRate) => {
 const getCriteriaAllowed = (criteriaNeeded, priceMaximum, maxDailyRate) =>
   getCriteriaNeeded(criteriaNeeded, priceMaximum, maxDailyRate) + 2
 
+const getMinimumMessage = (criteriaNeeded, essentialCriteria) => {
+  if (essentialCriteria.length > 0) {
+    return (
+      <span>
+        You must select at least {criteriaNeeded - essentialCriteria.length}{' '}
+        <strong> &apos;Other criteria&apos;</strong>
+      </span>
+    )
+  }
+
+  return (
+    <span>
+      You must submit evidence for at least {criteriaNeeded} {criteriaNeeded === 1 ? 'criterion' : 'criteria'}
+    </span>
+  )
+}
+
+const getMaximumMessage = (criteriaAllowed, criteriaNeeded, essentialCriteria) => {
+  if (essentialCriteria.length > 0) {
+    return (
+      <span>
+        You must select a maximum of {criteriaNeeded} <strong> &apos;Other criteria&apos;</strong>
```

Comment:
Do we need to factor in the number of essential criteria (eg criteriaAllowed - essentialCriteria.length)?